src/backbone_router/backbone_agent.hpp

```diff
@@ -106,9 +106,9 @@ class BackboneAgent : private NonCopyable
 
     static const char *StateToString(otBackboneRouterState aState);
 
-    otbr::Ncp::ControllerOpenThread &mNcp;
-    otBackboneRouterState            mBackboneRouterState;
-    Ip6Prefix                        mDomainPrefix;
+    otbr::Ncp::ControllerOpenThreadRcp &mCtrlr;
```

Comment:
I'd suggest to name it `mControllerRcp` or `mController`. From `mCtrlr` I can infer it is a controller but I didn't know what the ending `r` means.

Reply:
Make sense. I've updated them to `mController`.